configs/ssd300_mobilenetv2_coco.py

```diff
@@ -0,0 +1,129 @@
+# model settings
+input_size = 300
+model = dict(
+    type='SingleStageDetector',
+    pretrained='./snapshots/mobilenet_v2.pth.tar',
+    backbone=dict(
+        type='SSDMobilenetV2',
+        input_size=input_size,
+        activation_type='relu6'
+        ),
+    neck=None,
+    bbox_head=dict(
+        type='SSDHead',
+        input_size=input_size,
+        in_channels=(576, 1280, 512, 256, 256, 128),
+        num_classes=81,
+        anchor_strides=(16, 30, 60, 100, 150, 300),
+        basesize_ratio_range=(0.2, 0.95),
+        anchor_ratios=([2], [2, 3], [2, 3], [2, 3], [2], [2]),
+        target_means=(.0, .0, .0, .0),
+        target_stds=(0.1, 0.1, 0.2, 0.2),
+        depthwise_heads=True)
+        )
+cudnn_benchmark = True
+train_cfg = dict(
+    assigner=dict(
+        type='MaxIoUAssigner',
+        pos_iou_thr=0.5,
+        neg_iou_thr=0.5,
+        min_pos_iou=0.,
+        ignore_iof_thr=-1,
+        gt_max_assign_all=False),
+    smoothl1_beta=1.,
+    allowed_border=-1,
+    pos_weight=-1,
+    neg_pos_ratio=3,
+    debug=False)
+test_cfg = dict(
+    nms=dict(type='nms', iou_thr=0.45),
+    min_bbox_size=0,
+    score_thr=0.02,
+    max_per_img=200)
+# model training and testing settings
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+img_norm_cfg = dict(mean=[0, 0, 0], std=[255, 255, 255], to_rgb=True)
+data = dict(
+    imgs_per_gpu=70,
```

Comment:
For benchmarking, could you please provide a comparison with ssd300 baseline we provided? It would be great if the two configs share the same hyper-parameters if possible (lr, epoch).

Reply:
Ok, I'll preserve the number of epochs, but MobileNetV2 requires higher learning rate. It would be the only difference. Currently my best snapshot under these conditions gives 19.6 mAP vs 25.7 (ssd300).